### DISCUSSION TITLE
How to incorporate cookbook examples into the documentation

### DISCUSSION THREAD

**Discussion by bocklund** [pycalphad/pycalphad#324] [Ideas]
https://github.com/pycalphad/cookbook is starting to collect more examples of specific use cases and solutions for using pycalphad. It would be nice if these examples were more discoverable.

1. Should the website link to the cookbook or have cookbook examples incorporated directly? 

2. If the pycalphad website examples and the cookbook should remain separate, what is the criteria to determine whether an example belongs in the documentation examples vs. in the cookbook? 

Lets discuss this here and open an issue when we have some action items.

**Comment by MaxPrun** [comment]
Hi!

Thanks very much for the cookbook, as a new user this is extremly helpful. However I have a problem. I installed pycalphad in the version 0.9.2 via conda and tried your equilibrium calculation from the coolbook, but I'm getting a "ParseException: Expected" ... I see a similar issue has come up in your google group. Do you have a fix for that?

Thanks and cheers,
Max
PS: Sorry for the double posting, but I see the community on Github is much more active than on Google ;-)

![image](https://user-images.githubusercontent.com/95910810/145552409-0c26b4cc-60b2-4532-8d9d-f86480ea0bdc.png)

**Reply by richardotis** [reply]
Can you try one of the MC databases from the repository here: https://github.com/pycalphad/binder/tree/develop/multicomponent-databases
There was a subtle change in the parser not too long ago and it broke the pycalphad-compatible copies of the MC database that were floating around prior to October.
